guides/doc-Upgrading_and_Updating/topics/proc_synchronizing-smartproxy-servers-through-project-web-ui.adoc

```diff
@@ -0,0 +1,10 @@
+[id="proc_synchronizing-{smart-proxy-context}-servers-through-{project-context}-web-ui_{context}"]
+= Synchronizing {SmartProxyServer}s Through {Project} Web UI
```

Comment:
`{ProjectWebUI}`

Reply:
I'd prefer this one to be applied, otherwise non-blocking.

Reply:
I thought about this but then the resulting text would be "Satellite web UI" instead of "Satellite **W**eb UI"

Reply:
Sure thing.